config/routes.rb

```diff
@@ -3,4 +3,11 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+get "/customers/:id", to: "customers#show"
+
+post "/customers/:id/items", to: "customer_items#create"
```

Comment:
I do like this routing, the nesting shows the relationship between items and customers. routes all look restful, good job.